app/services/providers/copy_to_recruitment_cycle_service.rb

```diff
@@ -0,0 +1,67 @@
+module Providers
+  class CopyToRecruitmentCycleService
+    def initialize(provider:, copy_course_to_provider_service:, copy_site_to_provider_service:)
+      @provider = provider
+      @copy_course_to_provider_service = copy_course_to_provider_service
+      @copy_site_to_provider_service = copy_site_to_provider_service
+    end
+
+    def execute(new_recruitment_cycle)
```

Comment:
I'd expect the provider to be passed in to here rather than the initializer - then it's clear what provider you're copying to the service

Additionally we may want to copy multiple providers in sequence and having to reinitialise this should be a bit of a faff